shared/src/main/scala/org/jetbrains/sbt/structure/data.scala

```diff
@@ -128,15 +129,17 @@ case class ScalaData(organization: String,
   def allCompilerJars: Seq[File] = libraryJars ++ compilerJars
 }
 
-case class DependencyData(projects: Seq[ProjectDependencyData],
+case class DependencyData(projects: Dependencies[ProjectDependencyData],
                           modules: Seq[ModuleDependencyData],
                           jars: Seq[JarDependencyData])
 
+case class Dependencies[T](test: Seq[T], production: Seq[T])
```

Comment:
I did this change because I in the previous solution (with using `internalDependencyClasspath` task) 
I already had dependencies divided into test and production. For now it is not done yet, but this structure remained. I can remove it, but it will be useful in the future.

Reply:
Type parameter `[T]` confuses, taking that it's not actually used anywhere.
Everywhere in the code the value of the `T` is actually `ProjectDependencyData`.
While it's not needed I suggest inlining `ProjectDependencyData` into `Seq[T]`